### PR TITLE
fix(Decimal): parse integers as decimal. 

### PR DESCRIPTION
**Pull request by zbyte64**
* Accept IntNode into decimal type
* Add test exhibiting failure
* Enable py38 in tox

### CODE REVIEW

**Comment by ulgens** [comment]
@jkimbo :wave:

File: graphene/types/tests/test_decimal.py [hunk 1]
```diff
@@ -41,3 +41,11 @@ def test_bad_decimal_query():
...
     assert len(result.errors) == 1
     assert result.data is None
+
+
+def test_decimal_string_query_integer():
+    decimal_value = 1
+    result = schema.execute("""{ decimal(input: %s) }""" % decimal_value)
+    assert not result.errors
+    assert result.data == {"decimal": str(decimal_value)}
+    assert decimal.Decimal(result.data["decimal"]) == decimal_value
```
Comment (jkimbo) [inline graphene/types/tests/test_decimal.py:51]: For completeness can you add a test for a float value as well?

**Review by jkimbo — CHANGES_REQUESTED** [review]

File: graphene/types/tests/test_decimal.py (hunk 1 above)
Comment (zbyte64) [reply graphene/types/tests/test_decimal.py:51]: I don't think it's possible. So I'll remove the FloatValueNode instead

**Review by jkimbo — APPROVED** [review]

**Comment by ulgens** [comment]
@jkimbo Would you consider a new release for this?

**Comment by jkimbo** [comment]
@ulgens sure can do.

**Comment by jkimbo** [comment]
@ulgens published a new beta release: https://github.com/graphql-python/graphene/releases/tag/v3.0.0b7

Might take a while before it gets through Travis though.

**Pull request merged**